source/common/config/new_delta_subscription_state.cc

```diff
@@ -240,8 +222,7 @@ void NewDeltaSubscriptionState::handleGoodResponse(
                                           message.removed_resources());
   }
 
-  if (Runtime::runtimeFeatureEnabled(
-          "envoy.reloadable_features.delta_xds_subscription_state_tracking_fix")) {
+  {
```

Comment:
nit: I don't think the inner scope (curly brackets) is needed anymore.

Reply:
Done